src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -183,13 +178,11 @@ end
       Ja12_avg = 0.5 * (Ja12_node + Ja12_node_ii)
       # compute the contravariant sharp flux
       fluxtilde1 = Ja11_avg * flux1 + Ja12_avg * flux2
-      integral_contribution = alpha * derivative_split[i, ii] * fluxtilde1
-      add_to_node_vars!(du, integral_contribution, equations, dg, i,  j, element)
-      integral_contribution = alpha * derivative_split[ii, i] * fluxtilde1
-      add_to_node_vars!(du, integral_contribution, equations, dg, ii, j, element)
+      add_to_node_vars!(du, alpha * derivative_split[i, ii], fluxtilde1, equations, dg, i,  j, element)
```

Comment:
Could this be improved (at least visually, but maybe also in performance?) by adding multiple `multiply_and_add_to_node_vars!` that accept two or three arguments that get multiplied? This way, it would at least look consistent again...

Reply:
You mean something like
```julia
add_to_node_vars!(du, alpha, derivative_split[i, ii], fluxtilde1, ...)
```
I don't think this could increase the performance since there would only be multiple multiplications without intermediate additions that could be turned into an FMA. Additionally, constant propagation should hopefully be able to see that `alpha * derivative_split[i, ii] == derivative_split[i, ii]` when `alpha == true` (default value).

Reply:
OK, then what about consistency reasons? Now I have to know how which arguments to pass to this function and which ones I don't. Since we only have either 2 or 3 multiplication, would it hurt to add the 3 factors version as well and make it symmetric wrt the input arguments? Without knowing what's going on under the hood, the current implementation looks to me like there is a "special case" argument why `alpha` is multiplied to the first argument and not the second, and why they are passed in this order etc.

Reply:
Well, the signature is
```julia
multiply_add_to_node_vars!(u, factor, u_node, equations, solver::DG, indices...)
```
Thus, it appears reasonable to me that all scalar factors are passed together.

Reply:
Shall we also put this on the agenda?

Reply:
Sounds good!

Reply:
Decided: Leave it as it is